src/Vatras/Test/Test/VariantList-Completeness.agda

```diff
@@ -14,8 +15,8 @@ open import Vatras.Test.UnitTest
 
 open import Vatras.Framework.Variants using (Rose)
 Variant = Rose ∞
-import Vatras.Lang.VariantList
-open Vatras.Lang.VariantList Variant using (Configuration; ⟦_⟧; encode; vl-conf; vl-fnoc)
+open import Vatras.Lang.All.Fixed ⊥ Variant
```

Comment:
Nice trick. Maybe document shortly that for Clone-and-Own, no feature annotations exist, hence there is no feature annotation language. To prevent the use of any feature information, we use \bot here.